site/pom.xml

```diff
@@ -12,6 +12,10 @@
 		<version>0.8.0-SNAPSHOT</version>
 	</parent>
 
+	<properties>
+		<publishingPath>mars/snapshots/builds/${JOB_NAME}/${BUILD_ID}-B${BUILD_NUMBER}/all/repo/</publishingPath>
```

Comment:
for this to have effect the RedDeer job needs changing to reddeer to follow the conventions used everywhere else.